### PR TITLE
[luci] Copy-based quantization for weights

### PR DESCRIPTION
**Pull request by jinevening**
This supports copy-based quantization for weights.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: #5785

### CODE REVIEW

File: compiler/luci/pass/src/QuantizationUtils.h [hunk 1]
```diff
@@ -47,6 +47,40 @@ bool is_weights(CircleNode *node);
...
+    new_node->dim(i).set(node->dim(i).value());
+
+  const auto size = node->size<T>();
+  new_node->size<T>(size);
+  for (uint32_t i = 0; i < size; i++)
+    new_node->at<T>(i) = node->at<T>(i);
+
+  new_node->shape_status(luci::ShapeStatus::VALID);
+
+  auto qparam = std::make_unique<CircleQuantParam>();
```
Comment (seanshpark) [inline compiler/luci/pass/src/QuantizationUtils.h:72]: maybe do this and below only when `node->quantparam()` is not null?

Comment (seanshpark) [reply compiler/luci/pass/src/QuantizationUtils.h:72]: or if `node->quantparam()` be always exist then add `assert(node->quantparam() != nullptr);` at the top would be nice.

Comment (jinevening) [reply compiler/luci/pass/src/QuantizationUtils.h]: I've added the null check. Thanks.

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Pull request merged**